hw/ip/entropy_src/doc/_index.md

```diff
@@ -134,8 +134,14 @@ See the timing diagrams below for more details on how the health tests work.
 It should be noted that for all error counter registers, they are sized for 16 bits, which prevents any case where counters might wrap.
 
 
-The Mailbox test and Vendor-specific tests are planned, but not yet implemented.
-The theory of operation for these tests will therefore be described in a future update.
+The firmware override function has the capability to completely override the hardware health tests and the conditioner paths.
```

Comment:
Is the vendor-specific test still planned?

Reply:
Yes, diagram shows this. Added text to explain the interface in the doc section.

Reply:
Also, some dots were added to show more clarity on bus crossings.